api-admin/src/main/java/bio/terra/pearl/api/admin/service/logging/LoggingExtService.java

```diff
@@ -0,0 +1,20 @@
+package bio.terra.pearl.api.admin.service.logging;
+
+import bio.terra.pearl.core.model.log.LogEvent;
+import bio.terra.pearl.core.model.log.LogEventType;
+import bio.terra.pearl.core.service.LoggingService;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class LoggingExtService {
+  private final LoggingService loggingService;
+
+  public LoggingExtService(LoggingService loggingService) {
+    this.loggingService = loggingService;
+  }
+
```

Comment:
add an @SuperuserOnly annotation here

Reply:
I'm going to split the SuperuserOnly annotation out into it's own PR because it's critical auth-related code and I would prefer a very tightly scoped PR for that